### PR TITLE
Proj 3/home screen

### PR DESCRIPTION
**Pull request by kenzywa**
Created home screen with list of features

### CODE REVIEW

File: TableApplication/Feature/FeatureListViewController.swift [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import UIKit
+
+class HomeScreenViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    var homeScreenTableView = UITableView(frame: .zero, style: .plain)
+    var temporaryLabel = UILabel(frame: CGRect(x: 0, y: 0, width: 300, height: 15))
```
Comment (damencki) [inline TableApplication/Feature/FeatureListViewController.swift]: use autolayout instead frame

File: TableApplication/Feature/FeatureListViewController.swift [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    let identifier = "MyCell"
+    var features = [Feature(name: "List of Employees", task: "Show list of employees", id: UUID().uuidString)]
+    
+    private struct constants {
```
Comment (damencki) [inline TableApplication/Feature/FeatureListViewController.swift]: **C**onstants

File: TableApplication/Feature/FeatureListViewController.swift [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        static let heightOfRow : Float = 80.0
+        static let titleOfView : String = "List of Features"
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.title = constants.titleOfView
+        createTable()
+        view.addSubview(homeScreenTableView)
```
Comment (damencki) [inline TableApplication/Feature/FeatureListViewController.swift]: repetetive add tableview

File: TableApplication/Feature/FeatureListViewController.swift [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        temporaryLabel.center = CGPoint(x: 170, y: 150)
```
Comment (damencki) [inline TableApplication/Feature/FeatureListViewController.swift]: use autolayout

File: TableApplication/Feature/FeatureListViewController.swift [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+        temporaryLabel.textAlignment = .natural
+        temporaryLabel.font = UIFont(name: "San Francisco", size: 5.0)
+        temporaryLabel.text = features.first?.task
```
Comment (damencki) [inline TableApplication/Feature/FeatureListViewController.swift]: separate binding from layout

File: TableApplication/Feature/FeatureListViewController.swift [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+        temporaryLabel.textColor = .systemBlue
+        view.addSubview(temporaryLabel)
```
Comment (damencki) [inline TableApplication/Feature/FeatureListViewController.swift]: why label on the top of subviews?

**Review by damencki — CHANGES_REQUESTED** [review]

File: TableApplication/Feature/FeatureListViewController.swift (hunk 3 above)
Comment (kenzywa) [reply TableApplication/Feature/FeatureListViewController.swift]: Fixed

File: TableApplication/Feature/FeatureListViewController.swift (hunk 2 above)
Comment (kenzywa) [reply TableApplication/Feature/FeatureListViewController.swift]: Fixed

File: TableApplication/Feature/FeatureListViewController.swift (hunk 5 above)
Comment (kenzywa) [reply TableApplication/Feature/FeatureListViewController.swift]: Fixed

File: TableApplication/Feature/FeatureListViewController.swift (hunk 6 above)
Comment (kenzywa) [reply TableApplication/Feature/FeatureListViewController.swift]: Fixed

File: TableApplication/Feature/FeatureListViewController.swift (hunk 1 above)
Comment (kenzywa) [reply TableApplication/Feature/FeatureListViewController.swift]: Fixed

**Comment by kenzywa** [comment]
All issues fixed and solved

File: TableApplication/Feature/FeatureListViewController.swift (hunk 6 above)
Comment (damencki) [reply TableApplication/Feature/FeatureListViewController.swift]: Screenshot 2021-04-08 at 12 04 04 PM

File: TableApplication/Feature/FeatureListViewController.swift (hunk 5 above)
Comment (damencki) [reply TableApplication/Feature/FeatureListViewController.swift]: extract temporaryLabel.text = features.first?.task to method

**Review by damencki — CHANGES_REQUESTED** [review]

**Comment by kenzywa** [comment]
Solved

**Review by damencki — APPROVED** [review]

**Pull request merged**